src/main/java/org/terasology/lost/ProgressTrackingComponent.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.lost;
+
+import org.terasology.entitySystem.Component;
+import org.terasology.math.geom.Vector3i;
+import org.terasology.polyworld.graph.Region;
+
+import java.util.HashMap;
+
+public class ProgressTrackingComponent implements Component {
```

Comment:
Little concerned about the complexity in here, components are generally meant to just be dumb data containers. Have you seen if this saves & reloads right? Not sure if that's something we've tested much, and there are known issues, but usually if I see something non-basic it makes me worry it might run into weirdness with serialization.

For any sort of special stuff in components adding some javadoc should help as well 👍

Reply:
Tried saving and reload and the component works fine. Added javadoc in the next commit.